libr/core/agraph.c

```diff
@@ -2310,12 +2310,22 @@ static void add_child(RCore *core, RAGraph *g, RANode *u, ut64 jump) {
 	}
 	char *title = get_title (jump);
 	RANode *v = r_agraph_get_node (g, title);
-	ut64 a = r_num_get (NULL, u->title);
-	ut64 b = r_num_get (NULL, title);
+	if (v) {
+		ut64 a = r_num_get (NULL, u->title);
+		int len = 64;
+		char key[len];
```

Comment:
this is a variable size array, which is bad in many ways:

* not all c compilers support this
* can cause stack exhaustion if the value is big (in this case is constant, but the compiler doesnt needs to know that)

char do char key[64] and use sizeof(key) instead of len, this way the size of the array is always consistent with the array itself. not a separate variable

Reply:
Thanks, I did not know that. I have updated the code.